### PR TITLE
test: cover BlockingStorage and RxStorageWrapper

### PR DESCRIPTION
**Pull request by baudoliver7**
Test remaining functions of `BlockingStorage` and all functions of `RxStorageWrapper`

Close: #55
Ref: #383

### CODE REVIEW

**Comment by baudoliver7** [comment]
@artipie/maintainers ready to be reviewed :)

**Review by genryxy — CHANGES_REQUESTED** [review]
@baudoliver7 thanks, one suggestion

File: src/test/java/com/artipie/asto/blocking/BlockingStorageTest.java [hunk 1]
```diff
@@ -104,4 +107,49 @@ void shouldDeleteInOriginalWhenKeyIsDeletedByBlocking() throws Exception {
...
+                new Key.From("root2", "r2a"),
+                new Key.From("root3")
+            )
+        );
+        this.blocking.deleteAll(Key.ROOT);
+        MatcherAssert.assertThat(
+            "Original should not have any more item",
+            this.original.list(Key.ROOT).join().size(),
+            new IsEqual<>(0)
+        );
```
[the remark this answers is not in the document]
Comment (baudoliver7) [reply src/test/java/com/artipie/asto/blocking/BlockingStorageTest.java:153]: Ok

**Review by genryxy — APPROVED** [review]
@baudoliver7 thanks, looks good to me
@g4s8 please check

**Comment by baudoliver7** [comment]
@g4s8 how about this ?

**Pull request merged**